src/tasks/block.rs

```diff
@@ -155,12 +153,12 @@ impl BlockBuilder {
         }
     }
 
-    async fn _get_bundles(&mut self, in_progress: &mut InProgressBlock) {
+    async fn get_bundles(&mut self, in_progress: &mut InProgressBlock) {
         tracing::trace!("query bundles from cache");
         let bundles = self.bundle_poller.check_bundle_cache().await;
+        // OPTIMIZE: Sort bundles received from cache
```

Comment:
this is a good topic to discuss async/in other platform: what's gonna be the bundling strategy overall here? probably worth ticketing this.